### PR TITLE
Fix #1194 - Correct the flow in preview section.

### LINKED ISSUE #1194: Correct the flow in preview
**Actual Behaviour**

If user clicks on share from preview and then presses the back button, the preview window is closed and the image is saved.

**Expected Behaviour**

The preview should be active when the user returns.

**Steps to reproduce it**

1) Take a photo
2) Click on share in preview
3) Return back

**Screenshots of the issue**

21730483_1812068322155307_3557457214400626688_n

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by mayank8318**
Fix #1194 

Changes: Now the preview does not close upon returning from the Share section.

Screenshots for the change: 
21536022_1814180341944105_1673613107299090432_n

### CODE REVIEW

**Comment by mayank8318** [comment]
@vinaysajjanapu Ok

**Comment by mayank8318** [comment]
@vinaysajjanapu Please review.
Thanks!

File: app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java [hunk 1]
```diff
@@ -378,6 +380,17 @@ public void onClick(DialogInterface dialogInterface, int i) {
...
 
+    /**
+     * Method to send the result of the share operation
+     * @param code - SUCCESS if user shares it FAIL otherwise.
+     */
+    private void sendResult(int code) {
+        Intent result = new Intent();
+        result.putExtra(Constants.SHARE_RESULT, code);
+        setResult(RESULT_OK, result);
+        finish();
```
Comment (anantprsd5) [inline app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: This is finishing the share activity before the image is even shared. Just try to share an image to imgur.

Comment (mayank8318) [reply app/src/main/java/org/fossasia/phimpme/share/SharingActivity.java]: I need forgot to check for AsyncTasks. I made the corrections now.
Please review!
Thanks

**Pull request merged**